apps/web-giddh/src/app/accounting/journal-voucher/invoice/invoice.component.html

```diff
@@ -0,0 +1,231 @@
+<div class="col-xs-10">
+    <div class="col-xs-12 bg_wrap">
+        <div class="row">
+            <div class="clearfix pdT2">
+
+                <div class="d-flex">
+                    <span class="btn label isActive">{{data.voucherType}}</span>
+                    <input type="text" [(ngModel)]="data.invoiceNumberAgainstVoucher" [disabled]="!isCustomInvoice"
+                        name="invoiceNumberAgainstVoucher" placeholder="Invoice No" class="text-center appearance-none"
+                        style="width: 128px" />
+                </div>
+
+                <div class="pull-right text-right mrB1">
+                    <input type="text" onReturn #dateField [textMask]="{mask: dateMask, guide: true}"
+                        [(ngModel)]="entryDate" name="entryDate" (blur)="dateEntered();" placeholder="DD-MM-YYYY"
+                        class="form-control text-right invoice-date-field" /> {{ displayDay }}
+                </div>
+            </div>
+
+            <!-- invoice num -->
+            <div class="clearfix form-inline mrB">
+                <div class="mx-350 form-group" [hidden]="isSalesInvoiceSelected">
+                    <!-- decimalDigitsDirective -->
+                    <label>{{ invoiceNoHeading }}:</label> <input type="text" placeholder="Invoice no."
+                        [(ngModel)]="data.invoiceNumberAgainstVoucher" class="form-control max-height28 appearance-none"
+                        style="max-width:200px" [disabled]="isSalesInvoiceSelected" />
+                </div>
+            </div>
+            <!-- /invoice num -->
+
+
+            <!-- party acc name -->
+            <div class="mx-350 clearfix form-inline mrB">
+                <div class="form-group">
+                    <label>Party A/c Name: <sup>*</sup></label>
+                    <!-- #creditor -->
+                    <input #partyAccNameInputField name="party_name" type="text" placeholder="Party account"
+                        class="form-control max-height28 appearance-none upper-fields"
+                        (input)="onPartyAccFocusInput($event);selectedField='account';focusedField = 'partyAcc';"
+                        (keyup)="searchAccount($event?.target?.value)" (keydown)="detectKey($event)"
+                        [(ngModel)]="creditorAcc.name"
+                        (focus)="onPartyAccFocus($event);selectedField='account';focusedField = 'partyAcc';"
+                        (blur)="onPartyAccBlur();selectedField=null;focusedField = null;filterByText = ''"
+                        style="max-width:200px;" />
```

Comment:
remove inline css

Reply:
@rohit-walkover This code is copy-paste and work will be done in near future on this